foundations/javascript_basics/fundamentals-4.md

```diff
@@ -19,7 +19,7 @@ Strings and numbers may be our building blocks, but as your scripts get more com
 Computers don't get tired, and they're really, _really_ fast! For that reason, they are well suited to solving problems that involve doing calculations multiple times. In some cases, a computer will be able to repeat a task _thousands_ or even _millions_ of times in just a few short seconds where it might take a human many hours. \(Obviously, speed here depends on the complexity of the calculation and the speed of the computer itself\). One way to make a computer do a repetitive task is using a **loop**.
 
 1. Read this [MDN article](https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Looping_code). It's a longer one, but make sure you tackle the 'Active Learning' sections at the bottom of the page.
-2. Once again, same info, slightly different context from [JavaScript.info](http://javascript.info/while-for). \(Skim the info if you think you know it all, but **don't forget the tasks at the end of the page**. You learn best by _doing_.\)
+2. Once again, same info, slightly different context from [JavaScript.info](http://javascript.info/while-for). \(Skip the info if you think you know it all, but **don't forget the tasks at the end of the page**. You learn best by _doing_.\)
```

Comment:
I believe the intention here is for users to skim the content even if they feel they know the subject, sort of as a refresher, rather than skip the content just because they think they know the content.

We could tweak the verbiage to something along the lines of, "Be sure to read this content even if you feel like you have a good grasp on it." or something.